### PR TITLE
RichText: Documented missing parameters for isEmpty and isEmptyLine

### PR DESCRIPTION
**Pull request by javidalkaruzi**
See #22907.

## Description
Fixed the JSDoc warnings in  file packages/rich-text/src/is-empty.js

   6:1  warning  Missing JSDoc @param "value.text" declaration   jsdoc/require-param
  10:0  warning  Missing @param "value.text"                     jsdoc/check-param-names
  18:1  warning  Missing JSDoc @param "value.text" declaration   jsdoc/require-param
  18:1  warning  Missing JSDoc @param "value.start" declaration  jsdoc/require-param
  18:1  warning  Missing JSDoc @param "value.end" declaration    jsdoc/require-param
  22:0  warning  Missing @param "value.text"                     jsdoc/check-param-names
  22:0  warning  Missing @param "value.start"                    jsdoc/check-param-names
  22:0  warning  Missing @param "value.end"                      jsdoc/check-param-names

✖ 8 problems (0 errors, 8 warnings)
  0 errors and 4 warnings potentially fixable with the `--fix` option.
  
  
## How has this been tested?
npm run lint-js packages/rich-text/src/is-empty.js

## Types of changes
Bug fix

### CODE REVIEW

File: packages/rich-text/src/is-empty.js [hunk 1]
```diff
@@ -19,7 +21,11 @@ export function isEmpty( { text } ) {
  * Check if the current collapsed selection is on an empty line in case of a
  * multiline value.
  *
- * @param  {Object} value Value te check.
+ * @param {Object}      value           Object containing the value that
+ *                                      should be checked.
+ * @param {string}      value.text      The text that should be check.
```
Comment (ellatrix) [inline packages/rich-text/src/is-empty.js:26]:
```suggestion
 * @param {string}      value.text      The text that should be checked.
```

File: packages/rich-text/src/is-empty.js [hunk 2]
```diff
@@ -4,12 +4,14 @@
...
 
 /**
- * Check if a Rich Text value is Empty, meaning it contains no text or any
+ * Check if a Rich Text value is empty, meaning it contains no text or any
  * objects (such as images).
  *
- * @param {Object} value Value to use.
+ * @param {Object}      value           Object containing the value that
+ *                                      should be checked.
+ * @param {string}      value.text      The text that should be check.
```
Comment (ellatrix) [inline packages/rich-text/src/is-empty.js:12]:
```suggestion
 * @param {string}      value.text      The text that should be checked.
```

File: packages/rich-text/README.md [hunk 3]
```diff
@@ -183,12 +183,13 @@ _Returns_
...
 
-Check if a Rich Text value is Empty, meaning it contains no text or any
+Check if a Rich Text value is empty, meaning it contains no text or any
 objects (such as images).
 
 _Parameters_
 
--   _value_ `Object`: Value to use.
+-   _value_ `Object`: Object containing the value that should be checked.
+-   _value.text_ `string`: The text that should be check.
```
Comment (ellatrix) [inline packages/rich-text/README.md:192]:
```suggestion
-   _value.text_ `string`: The text that should be checked.
```

File: packages/rich-text/src/is-empty.js [hunk 4]
```diff
@@ -4,12 +4,14 @@
 import { LINE_SEPARATOR } from './special-characters';
 
 /**
- * Check if a Rich Text value is Empty, meaning it contains no text or any
+ * Check if a Rich Text value is empty, meaning it contains no text or any
  * objects (such as images).
  *
- * @param {Object} value Value to use.
+ * @param {Object}      value           Object containing the value that
```
Comment (ellatrix) [inline packages/rich-text/src/is-empty.js:10]: Why increase the spacing here?

File: packages/rich-text/src/is-empty.js [hunk 5]
```diff
@@ -4,12 +4,14 @@
...
+ *                                      should be checked.
+ * @param {string}      value.text      The text that should be checked.
  *
- * @return {boolean} True if the value is empty, false if not.
+ * @return {boolean}    True if the value is empty, false if not.
```
Comment (ellatrix) [inline packages/rich-text/src/is-empty.js:14]: For return, there's no need to add extra spacing, just one is enough.

**Comment by gziolo** [comment]
Thank you @javidalkaruzi for your contribution. It looks like those issues were resolved in different PR. I'm closing this one.

**Pull request closed without merging**